satpy/readers/fci_l1c_nc.py

```diff
@@ -161,7 +165,7 @@ class using the :mod:`~satpy.Scene.load` method with the reader
 
     def __init__(self, filename, filename_info, filetype_info):
         """Initialize file handler."""
-        super(FCIFDHSIFileHandler, self).__init__(filename, filename_info,
+        super(FCIL1CNCFileHandler, self).__init__(filename, filename_info,
```

Comment:
We support only Python 3 now, so this can be simplified to `super().__init__(...)`.